metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -393,8 +358,18 @@ def new_dataset_or_report(tile_instance: Any) -> dict:
                 title=instance.get(Constant.TITLE),
                 embedUrl=instance.get(Constant.EMBED_URL),
                 dataset_id=instance.get(Constant.DATASET_ID),
+                report_id=instance.get(Constant.REPORT_ID),
                 dataset=None,
-                **new_dataset_or_report(instance),
```

Comment:
thanks for cleaning this up

are there any entries in `Constant` that can be removed now?